### PR TITLE
Gizmo docs

### PR DESCRIPTION
**Pull request by TrevorDev**
https://github.com/BabylonJS/Babylon.js/issues/4141
https://github.com/BabylonJS/Documentation/issues/826
https://github.com/BabylonJS/Documentation/issues/1022

### CODE REVIEW

File: content/How_To/mesh/UtilityLayerRenderer.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+PG_TITLE: Use UtilityLayerRenderer
+---
+
```
Comment (deltakosh) [inline content/How_To/mesh/UtilityLayerRenderer.md:4]: These behaviors must be referenced in the global behavior page

File: content/How_To/mesh/Gizmo.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Introduction
+
+Gizmo's are objects that can be attached to a mesh to provide interaction.
+
+## Setup
+
+Gizmo's are displayed by a [UtilityLayerRenderer](/How_To/UtilityLayerRenderer) to not disrupt the existing scene state.
+```
+var utilLayer = new BABYLON.UtilityLayerRenderer(scene);
+var gizmo = new BABYLON.AxisDragGizmo(utilLayer, new BABYLON.Vector3(1,0,0), BABYLON.Color3.FromHexString("#00b894"));
```
Comment (deltakosh) [inline content/How_To/mesh/Gizmo.md]: Can we invert the parameters and create a default utility layer if none is provided
The default utility layer should be shared between gizmos when no utility layer is defined

File: content/How_To/mesh/Gizmo.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+gizmo.updateGizmoRotationToMatchAttachedMesh = false;
+gizmo.updateGizmoPositionToMatchAttachedMesh = true;
+```
+[**Basic example**](https://www.babylonjs-playground.com/#31M2AP#3)
+
+## Position, scale and rotation gizmos
+
+Default gizmos for position, rotation and scale on a single axis are supported
+
+ - [Single Axis Position](https://www.babylonjs-playground.com/#31M2AP#3)
```
Comment (deltakosh) [inline content/How_To/mesh/Gizmo.md]: The API should also be mentioned here for search indexation purpose

File: content/How_To/mesh/Gizmo.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+```
+gizmo.snapDistance = 0.3;
+gizmo.onSnapObservable.add((event)=>{
+    console.log(event);
+})
+```
+
+Classes for 3 axis gizmos are also provided which contain 3 of the single axis gizmos within 
+
+ - [Multi Axis Position](https://www.babylonjs-playground.com/#31M2AP#6)
```
Comment (deltakosh) [inline content/How_To/mesh/Gizmo.md]: Same here

**Review by deltakosh — CHANGES_REQUESTED** [review]
Generally speaking we need that all properties and methods are mentioned here

File: content/How_To/mesh/Gizmo.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+```
+gizmo.snapDistance = 0.3;
+gizmo.onSnapObservable.add((event)=>{
+    console.log(event);
+})
+```
+
+Classes for 3 axis gizmos are also provided which contain 3 of the single axis gizmos within 
+
+ - [PositionGizmo](https://www.babylonjs-playground.com/#31M2AP#6)
```
Comment (TrevorDev) [inline content/How_To/mesh/Gizmo.md:47]: All these playground need to be updated. Waiting for https://github.com/BabylonJS/Babylon.js/pull/4606

Comment (deltakosh) [reply content/How_To/mesh/Gizmo.md:47]: merged and published

**Review by deltakosh — APPROVED** [review]

**Comment by deltakosh** [comment]
Perfect.. Please socialize it with the community :)

**Pull request merged**